devtools/optimize.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+function cleanup {
+  for folder in accessories/*
+  do
+    folder=${folder/'accessories/'}
+    rm -R contracts/$folder
+  done
+}
+
+trap cleanup EXIT
```

Comment:
your bash skills are so good

Reply:
i think it would be nice to add some comments to this file. maybe just one at the top being like "does this, then this, then this, because this".